Chiazokam/Politico
# #163504419 create the get all offices endpoint

#### What does this PR do?
This PR tests and creates the get all offices Endpont

#### Description of Task to be completed?
- Write tests for all functions
- Create the get all offices endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-get-all-offices-163504419`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `GET` request to `http://localhost:3000/api/v1/offices`
- You should receive a successful response

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163504419](https://www.pivotaltracker.com/story/show/163504419)

### Images
![image](https://user-images.githubusercontent.com/26940294/51886122-b7f31480-238e-11e9-9cd6-0b7a3e001bad.png)
